src/ipyaladin/widget.py

```diff
@@ -955,6 +975,34 @@ def add_graphic_overlay_from_stcs(
             }
         )
 
+    @widget_should_be_loaded
+    def remove_overlay(self, overlay_name: Union[Iterable[str], str]) -> None:
+        """Remove an overlay layer defined by a string.
+
+        Parameters
+        ----------
+        overlay_name : str, Iterable[str]
+            The string or an iterable of strings.
+
+        """
+        overlay_name = [overlay_name] if isinstance(overlay_name, str) else overlay_name
+
+        self.send(
+            {
+                "event_name": "remove_overlay",
+                "name": overlay_name,
+            }
+        )
+
+    @widget_should_be_loaded
+    def get_overlays(self) -> List:
```

Comment:
question/concern: Should this method be returning overlays? And if it does, maybe it should be a private method called by the `overlays` property to update the overlays before returning them to the user. I would be confused as a user if I had to juggle calling `aladin.overlays` and `aladin.get_overlays()